app/mailers/app_mailer.rb

```diff
@@ -1,8 +1,8 @@
 class AppMailer < ActionMailer::Base
   default from: "info@howling-hematite.herokuapp.com/"
 
-  def send_account_token(giver)
+  def receipt(giver, donation)
```

Comment:
Unused method argument - `donation`. If it's necessary, use `_` or `_donation` as an argument name to indicate that it won't be used.